packages/react/src/ReactLazy.js

```diff
@@ -98,6 +108,14 @@ function lazyInitializer<T>(payload: Payload<T>): T {
               // Mark the end time of when we resolved.
               // $FlowFixMe[cannot-write]
               ioInfo.end = performance.now();
+              // Surface the default export as the resolved "value" for debug purposes.
+              const debugValue =
+                moduleObject == null ? undefined : moduleObject.default;
+              resolveDebugValue(debugValue);
+              // $FlowFixMe
+              ioInfo.value.status = 'fulfilled';
+              // $FlowFixMe
+              ioInfo.value.value = debugValue;
```

Comment:
This block isn't gated on `enableAsyncDebugInfo` but `resolveDebugValue` is only callable with `enableAsyncDebugInfo`. `enableAsyncDebugInfo` isn't enabled everywhere at the moment.

Reply:
I guess this is implied via `payload._ioInfo` but still nice for DCE and readability to gate it.